database/schema.sql

```diff
@@ -35,6 +35,13 @@ CREATE TABLE trades (
     PRIMARY KEY (block_number, log_index)
 );
 
+CREATE TABLE invalidations (
```

Comment:
ubernit: In the future, we might want to support offchain-invalidations as well. Do you think those would be stored in the same database? Otherwise we can maybe already call this table invalidation_events or something else hinting at the fact that this is onchain?

Reply:
Offchain invalidation would be the delete orders route, right? In that case I thought we would delete the order directly from the database. I don't want to store those in this table because if it's not an event it has a different schema. I'll add a comment in the schema to explain what the table is.

Reply:
I think we will at some point need "time-travel" capabilities to derive the state of the orderbook at a certain point in time (maybe block number) so that we can reason about the "fairness" of solutions (and/or benchmark changes based on historic data). Therefore, I'd not remove anything from the table but just add at what point in time an order became invalid and maybe also store a timestamp with each event/block number in order to be able to join off-chain and on-chain events.

But it's not something we need to tackle right now.